assets/js/components/Collection/Form.jsx

```diff
@@ -51,9 +50,11 @@ const CollectionForm = ({ collection }) => {
     CREATE_COLLECTION,
     {
       onCompleted({ createCollection }) {
-        addToast(`Collection ${createCollection.name} created successfully`, {
-          appearance: "success",
-          autoDismiss: true
+        toast({
+          message: `Collection ${createCollection.name} created successfully`,
+          type: "is-success",
+          dismissible: true,
+          duration: 5000
         });
```

Comment:
@katdivyareddy10 Since we're repeating this same block of code everywhere, I'd like it to be a custom UI component.  The only thing the component should accept as an argument is `type` and `message`.

Reply:
I added a vanilla JS wrapper for this.